com.stremio.Stremio.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```

Comment:
Ideally you should maintain it in https://github.com/Stremio/stremio-shell but can be handled after the review.

Reply:
We have another Desktop file in the repo but it expects the shell to be in /opt/stremio. Is it better if I patch the mainstream Desktop file?

Reply:
Yeah, for example you can also use `desktop-file-edit` to modify specific keys.

Reply:
Thanks. I'll change it.